app/controllers/questionnaires_controller.rb

```diff
@@ -3,6 +3,7 @@ class QuestionnairesController < ApplicationController
 
   def show
     questionnaire_taker.prepare
+    @module_item = ModuleItem.find_by(training_module: module_params["training_module_id"], name: module_params["id"])
```

Comment:
Did you say `@questionnaire.module_item` wasn't always happy?

Reply:
No, just back when we had 3 different controllers